x-pack/test_serverless/functional/test_suites/observability/cypress/e2e/navigation.cy.ts

```diff
@@ -6,12 +6,11 @@
  */
 
 describe('Serverless', () => {
-  it('Should login', () => {
+  beforeEach(() => {
```

Comment:
Added this before each to avoid login in for every test

Reply:
Personally I don't mind having things duplicated. It is sometimes easier to read. But using the `beforeEach` hook is also just.